<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="37410" started-at="2016-08-09T07:17:36Z" finished-at="2016-08-09T07:18:13Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="37410" started-at="2016-08-09T07:17:36Z" finished-at="2016-08-09T07:18:13Z">
      <class name="com.maax.businessmanager.common.Certificate_Add">
        <test-method status="FAIL" signature="certificateadd(java.util.Hashtable)[pri:0, instance:com.maax.businessmanager.common.Certificate_Add@66d33a]" name="certificateadd" duration-ms="35678" started-at="2016-08-09T04:17:37Z" data-provider="SuiteCommontestdata" finished-at="2016-08-09T04:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{beneficary_relation=Self, policy=09420 (MEDAVIE), Certificate_number=000000003, Class=01: Sample class description, postal_code=T1C 1E2, Password=Welcome1, Username=bcchase, Member_LastName=Romganaikaloo, City=Lake City, Subdivision=, Adress_type=Home Address, Address_line_1=Temple Street, Member_DOB=01-05-1972, RiskFactor=Risk 1 - YM, Sal_amount=4500.0, Division=001, Employment_type=Full Time Salary, hiredate=01-04-2013, job_titile=SYS ANALYST, Income_freq=Monthly, Plan=0000003827: All Benefits, Province=Alberta, Number_of_hours=40.0, beneficiary_ls=HSTESTCMTESTWOZ, Designated date=01-01-2000, Runmode=Y, Province_of_Billing=Alberta, Country=Canada, correspondence_laun=English, Occupation_code=YA-Construction, beneficary_percentage=100.0, Column_to_search=Group Legal Name, Result=, Application_date=01-04-2013, Group_Legal_name=NKMBCGRP, Member_Gender=Male, number_of_units=10.0, Family_Category=Single, Member_FirstName=Margaret-Catherine, Browser=Firefox, Income_type=Salary}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[Element is not clickable at point (785.2833251953125, 516.1000061035156). Other element would receive the click: <td></td>
Command duration or timeout: 69 milliseconds
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'LMTN202461', ip: '10.15.2.63', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 54e7d578-da13-4fa6-bec8-13d078f7e57b]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: Element is not clickable at point (785.2833251953125, 516.1000061035156). Other element would receive the click: <td></td>
Command duration or timeout: 69 milliseconds
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'LMTN202461', ip: '10.15.2.63', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 54e7d578-da13-4fa6-bec8-13d078f7e57b
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at com.maax.businessmanager.util.TestBase.click(TestBase.java:91)
	at com.maax.businessmanager.common.Certificate_Add.certificateadd(Certificate_Add.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- certificateadd -->
        <test-method status="SKIP" signature="certificateadd(java.util.Hashtable)[pri:0, instance:com.maax.businessmanager.common.Certificate_Add@66d33a]" name="certificateadd" duration-ms="241" started-at="2016-08-09T04:18:13Z" data-provider="SuiteCommontestdata" finished-at="2016-08-09T04:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{beneficary_relation=Self, policy=09420 (MEDAVIE), Certificate_number=000000004, Class=01: Sample class description, postal_code=V1C 1E2, Password=Welcome1, Username=bcchase, Member_LastName=Saulnier Levesque, City=Vancouver, Subdivision=, Adress_type=Home Address, Address_line_1=Temple Street, Member_DOB=01-05-1976, RiskFactor=Risk 1 - YM, Sal_amount=4500.0, Division=001, Employment_type=Full Time Salary, hiredate=01-05-2013, job_titile=SYS ANALYST, Income_freq=Monthly, Plan=0000003827: All Benefits, Province=British Columbia, Number_of_hours=40.0, beneficiary_ls=HSTESTCMTESTTHREEZ, Designated date=01-01-2000, Runmode=N, Province_of_Billing=British Columbia, Country=Canada, correspondence_laun=English, Occupation_code=YA-Construction, beneficary_percentage=100.0, Column_to_search=Group Legal Name, Result=, Application_date=01-05-2013, Group_Legal_name=NKMBCGRP, Member_Gender=Male, number_of_units=10.0, Family_Category=Single, Member_FirstName=Christabelladoux, Browser=Firefox, Income_type=Salary}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is configured as No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is configured as No
	at com.maax.businessmanager.util.TestBase.checkrunmodes(TestBase.java:79)
	at com.maax.businessmanager.util.TestBase.init(TestBase.java:56)
	at com.maax.businessmanager.common.Certificate_Add.certificateadd(Certificate_Add.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- certificateadd -->
        <test-method status="SKIP" signature="certificateadd(java.util.Hashtable)[pri:0, instance:com.maax.businessmanager.common.Certificate_Add@66d33a]" name="certificateadd" duration-ms="182" started-at="2016-08-09T04:18:13Z" data-provider="SuiteCommontestdata" finished-at="2016-08-09T04:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{beneficary_relation=Self, policy=09420 (MEDAVIE), Certificate_number=000000005, Class=01: Sample class description, postal_code=L1C 1E2, Password=Welcome1, Username=bcchase, Member_LastName=Happy-New-Year-Too, City=Toronto, Subdivision=, Adress_type=Home Address, Address_line_1=Temple Street, Member_DOB=01-04-1973, RiskFactor=Risk 1 - YM, Sal_amount=4500.0, Division=001, Employment_type=Full Time Salary, hiredate=01-04-2013, job_titile=SYS ANALYST, Income_freq=Monthly, Plan=0000003827: All Benefits, Province=Ontario, Number_of_hours=40.0, beneficiary_ls=HSTESTCMTESTFOURZ, Designated date=01-01-2000, Runmode=N, Province_of_Billing=Ontario, Country=Canada, correspondence_laun=English, Occupation_code=YA-Construction, beneficary_percentage=100.0, Column_to_search=Group Legal Name, Result=, Application_date=01-04-2013, Group_Legal_name=NKMBCGRP, Member_Gender=Male, number_of_units=10.0, Family_Category=Single, Member_FirstName=Feliz-Navidad, Browser=Firefox, Income_type=Salary}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is configured as No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is configured as No
	at com.maax.businessmanager.util.TestBase.checkrunmodes(TestBase.java:79)
	at com.maax.businessmanager.util.TestBase.init(TestBase.java:56)
	at com.maax.businessmanager.common.Certificate_Add.certificateadd(Certificate_Add.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- certificateadd -->
        <test-method status="SKIP" signature="certificateadd(java.util.Hashtable)[pri:0, instance:com.maax.businessmanager.common.Certificate_Add@66d33a]" name="certificateadd" duration-ms="165" started-at="2016-08-09T04:18:13Z" data-provider="SuiteCommontestdata" finished-at="2016-08-09T04:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{beneficary_relation=Self, policy=09420 (MEDAVIE), Certificate_number=000000005, Class=01: Sample class description, postal_code=A1C 1E2, Password=Welcome1, Username=bcchase, Member_LastName=Trenholm, City=Lake city, Subdivision=, Adress_type=Home Address, Address_line_1=Temple Street, Member_DOB=01-07-1970, RiskFactor=Risk 1 - YM, Sal_amount=4500.0, Division=001, Employment_type=Full Time Salary, hiredate=01-04-2013, job_titile=SYS ANALYST, Income_freq=Monthly, Plan=0000003827: All Benefits, Province=NFLD & Labrador, Number_of_hours=40.0, beneficiary_ls=HSTESTCMTESTFOURZ, Designated date=01-01-2000, Runmode=N, Province_of_Billing=NFLD & Labrador, Country=Canada, correspondence_laun=English, Occupation_code=YA-Construction, beneficary_percentage=100.0, Column_to_search=Group Legal Name, Result=, Application_date=01-04-2013, Group_Legal_name=NKMBCGRP, Member_Gender=Male, number_of_units=10.0, Family_Category=Single, Member_FirstName=Jasper, Browser=Firefox, Income_type=Salary}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is configured as No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is configured as No
	at com.maax.businessmanager.util.TestBase.checkrunmodes(TestBase.java:79)
	at com.maax.businessmanager.util.TestBase.init(TestBase.java:56)
	at com.maax.businessmanager.common.Certificate_Add.certificateadd(Certificate_Add.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- certificateadd -->
        <test-method status="SKIP" signature="certificateadd(java.util.Hashtable)[pri:0, instance:com.maax.businessmanager.common.Certificate_Add@66d33a]" name="certificateadd" duration-ms="217" started-at="2016-08-09T04:18:13Z" data-provider="SuiteCommontestdata" finished-at="2016-08-09T04:18:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{beneficary_relation=Self, policy=09420 (MEDAVIE), Certificate_number=000000005, Class=01: Sample class description, postal_code=E1C 1E2, Password=Welcome1, Username=bcchase, Member_LastName=Maxwell-Robbins, City=Moncton, Subdivision=, Adress_type=Home Address, Address_line_1=Temple Street, Member_DOB=01-04-1973, RiskFactor=Risk 1 - YM, Sal_amount=4500.0, Division=001, Employment_type=Full Time Salary, hiredate=02-04-2013, job_titile=SYS ANALYST, Income_freq=Monthly, Plan=0000003827: All Benefits, Province=New Brunswick, Number_of_hours=40.0, beneficiary_ls=HSTESTCMTESTFOURZ, Designated date=01-01-2000, Runmode=N, Province_of_Billing=New Brunswick, Country=Canada, correspondence_laun=English, Occupation_code=YA-Construction, beneficary_percentage=100.0, Column_to_search=Group Legal Name, Result=, Application_date=02-04-2013, Group_Legal_name=NKMBCGRP, Member_Gender=Male, number_of_units=10.0, Family_Category=Single, Member_FirstName=Bobbi-Jo , Browser=Firefox, Income_type=Salary}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is configured as No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is configured as No
	at com.maax.businessmanager.util.TestBase.checkrunmodes(TestBase.java:79)
	at com.maax.businessmanager.util.TestBase.init(TestBase.java:56)
	at com.maax.businessmanager.common.Certificate_Add.certificateadd(Certificate_Add.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- certificateadd -->
      </class> <!-- com.maax.businessmanager.common.Certificate_Add -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
